### ISSUE TITLE
Cannot create an item in a locked collection

### ISSUE THREAD

**Issue by yoderme** [Noovolari/leapp#256]
I was trying to set up Leapp - first time user.  We're using AWS SSO tied to Okta.  I tried to set up an AWS SSO integration and went through the steps - went to AWS, went to Okta, logged in, and it appeared to work - and then I got a red box with the message

"Error during SSO Login: Error: Cannot create an item in a locked collection"

No clue what that means.

**Leapp Version**
0.11

**To Reproduce**
See above

**Expected behavior**
No error

**Screenshots**
-

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Leapp Version 0.11

**Comment by ericvilla** [comment]
Hi @yoderme, thank you for reporting! 

When logging into AWS SSO integration, Leapp stores the AWS SSO access token - used to retrieve the list of accounts you can access and generate short-lived credentials - in the system keychain. 

Leapp asks you to unlock the system keychain - the login one, since we're not using a dedicated one - to store the AWS SSO access token; probably you have not unlocked it. Please, verify it on your side, thank you!

**Comment by yoderme** [comment]
Yeah I don't know what happened, but a reboot made that problem go away - looks like the login process unlocked the system keychain.  I had just installed the dependencies and installed leapp without a reboot.

Now my next strange problem.  On startup I see

----
$ leapp
[22007:0421/171646.216381:ERROR:gpu_init.cc(453)] Passthrough is not supported, GL is swiftshader, ANGLE is 
Checking for update

And then it exits.
----

My setup is a little strange. This is an ubuntu 20 virtual machine running in hyper-v on windows, using cygwin-x to create xwindows.

What's odd is that it _was_ starting up fine, then something happened (I don't know what) and then I got stuck on the above.

I also might as well say that I had uninstalled firefox because of something else, and leapp does not behave well when there's no web browser on the system.  When I use `aws sso login` I cut and paste things into chrome on my windows desktop and that's fine for me.

**Comment by ericvilla** [comment]
Hi @yoderme!

I guess you're using the AWS SSO integration's in-browser auth method. Have you already tried the in-app mode? 

By the way, I think we've to consider providing the verification URL to the user - as you suggested - so that the user can copy and paste it into the browser.
